sphinxcontrib/confluencebuilder/publisher.py

```diff
@@ -464,6 +471,18 @@ def __init__(self, server_url, proxy=None,
         self.https = (self.scheme == 'https')
         self.proxy = None
         self.timeout = timeout
+        if isinstance(client_cert, tuple) and len(client_cert) >= 2:
+            if len(client_cert) < 2:
+                self._certfile = client_cert[0]
+                self._keyfile = None
+            else:
+                self._certfile = client_cert[0]
```

Comment:
I would use native Python unpacking for this, eg. self._certfile, self._keyfile = client_cert